src/styles/components/application-preloader/styles.less

```diff
@@ -53,7 +53,8 @@
 
   .application-preloader-spinner {
     animation: application-preloader-spinner 1s linear infinite;
-    background-image: url('../../../img/components/application-preloader/application-preloader-spinner.png');
```

Comment:
Sad that we don't use postcss, there are nice ways to inline images on compile time. 🤷‍♂️